src/views/tables/FillsTable.tsx

```diff
@@ -313,12 +315,12 @@ export const FillsTable = ({
   const dispatch = useDispatch();
   const { isMobile, isTablet } = useBreakpoints();
 
-  const marketFills = useSelector(getCurrentMarketFills, shallowEqual) || [];
-  const allFills = useSelector(getSubaccountFills, shallowEqual) || [];
+  const marketFills = useSelector(getCurrentMarketFills, shallowEqual) ?? EMPTY_ARR;
```

Comment:
do we want all of these to share the same `EMPTY_ARR` object? is there a possibility that one of these vars gets updated and updates all of the other vars as a result?

Reply:
That would be bad, but state slices should all be immutable so I'm not super worried.

Reply:
To prevent issues we can Object.freeze the array actually. I'll add it